standards-catalogue/lib/schemaorg.rb

```diff
@@ -0,0 +1,84 @@
+module SchemaOrg
+  class Base
+    attr_accessor :obj
+
+    # Middleman uses ActiveSupport's JSON encoder, which requires `as_json` to produce the hash representation of the object
+    def as_json(_options = {})
+      @obj
+    end
+  end
+
+  class Standard < SchemaOrg::Base
+    def initialize(data, page_data)
+      super()
+
+      categories = categories(data, page_data)
+      licence_info = SchemaOrg::Licence.new(data, page_data.licence_id)
+      maintainer_info = SchemaOrg::Organisation.new(data, page_data.maintainer_id)
+      endorsement_status = page_data.endorsement_status_events.last.status
+      specification = SchemaOrg::Specification.new(page_data.name, page_data.specification_url)
+
+      @obj = {
+        "@context": "https://schema.org/",
+        "@type": "Thing",
+        "name": page_data.name,
+        "contraction": page_data.contraction,
```

Comment:
more of a question than a comment - do these things have to be schema.org properties, or can they be anything? Wondering if there's a type for contraction, e.g. [DefinedTerm](https://schema.org/DefinedTerm), or does it not really matter?

Reply:
I think they _should_ be schema.org things, but if not, it's not as much of a problem.

As we're using `Thing`, [schema.org validation](https://validator.schema.org/) warns us that i.e. `category` isn't a valid property, but it shouldn't break anything.

I think `DefinedTerm` could be a good fit for this - but we'd need to leave out the `inDefinedTermSet` because we don't currently have a page defining what each of these are